### PR TITLE
Giant Fruit item effect

### PR DESCRIPTION
**Pull request by AngieDutra**
THIS PR DEPENDS ON BACKEND BRANCH https://github.com/lambdaclass/mirra_backend/pull/726

## Motivation
Our giant fruit item was missing it's effect

## Summary of changes
This PR adds the giant fruit vfx and scale animation to the game
More detailed:
- Adds the giant vfx with fade in fade out animation
- Adds a scale animation according to the player radius from backend
- Adds a refactor to improve how we handle the effects in the client
- Adds logic to preserve the aim direction scale even though the player scales up and down

## How has this been tested?
Use [backend branch](https://github.com/lambdaclass/mirra_backend/pull/726)
Try it out in the items scene or just by playing the game
I recommend locally deleting the other items from the back so every item you pick is the giant. Also, I would decrease the interval for item spawns

## Checklist
- [x] I have tested the changes locally.
- [x] I have tested the whole game after applying the changes, not only the affected areas.
- [x] I self-reviewed the changes on GitHub, line by line.
- [x] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [x] Tested in Android.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Comment by AngieDutra** [comment]
I'll fix multiple player giant fruit effect since now you can only see the scaling animation for the client

**Review by tkz00 — CHANGES_REQUESTED** [review]
When other players become small after the fruit's effect has worn out, they get the yellow glow, but not for my own character.

https://github.com/lambdaclass/champions_of_mirra/assets/67983386/08e33eea-ee07-4ef5-9fd7-6918d174eaad

**Comment by alfopisano** [comment]
On the opposite side, I did not experience that Issue playing on my iOS:
https://github.com/lambdaclass/champions_of_mirra/assets/116388676/e465dcda-ff68-4957-9aae-82f7bf341859

File: client/Assets/Scripts/Effects/EffectCharacterMaterialController.cs [hunk 1]
```diff
@@ -13,6 +13,9 @@ public class EffectCharacterMaterialController : MonoBehaviour
     private void Start()
     {
         // print(character_material_manager);
+        /// FOR TEST POPOUSES ONLY, DO NOT SHIP
```
Comment (tkz00) [inline client/Assets/Scripts/Effects/EffectCharacterMaterialController.cs:16]: Should this lines be removed?

**Review by tkz00 — CHANGES_REQUESTED** [review]

**Review by tkz00 — DISMISSED** [review]
Great work! Looks amazing, tested on my android!

File: client/Assets/Scripts/Effects/EffectCharacterMaterialController.cs (hunk 1 above)
Comment (AngieDutra) [reply client/Assets/Scripts/Effects/EffectCharacterMaterialController.cs:16]: Yes, forgot to delete the comment and the print, I'll do it!

**Review by tkz00 — APPROVED** [review]
Great! Works on my android!

**Review by tvillegas98 — APPROVED** [review]

**Pull request merged**